ios/FluentUI/Label/BadgeLabel.swift

```diff
@@ -23,21 +23,26 @@ class BadgeLabel: UILabel {
     /// Base function for initialization
     private func initBase() {
         layer.masksToBounds = true
-        backgroundColor = Colors.Palette.dangerPrimary.color
-        textColor = .white
         textAlignment = .center
         font = UIFont.systemFont(ofSize: Constants.badgeFontSize, weight: .regular)
         isHidden = true
+
+        updateColors()
     }
 
     override func didMoveToWindow() {
```

Comment:
Do we need this function anymore since we're not using the window to update colors?

Reply:
Different windows can have different fluent themes, so the colors might still change between windows. So I think we should still have this, and we would probably just add it back when we tokenize the control.

Reply:
Agree with huwilkes - I think it's worth keeping this function!

Reply:
The compromise is to add themeDidChange alongside didMoveToWindow.